persistence/msmq/subscription.md

```diff
@@ -7,7 +7,9 @@ related:
  - samples/msmq/persistence
 ---
 
+Note: The subscription queue can contain duplicate entries and is by design. This does not result in events to be published multiple times. After a publisher restarts the subscription queue state will be rewritten and deduplicated.
```

Comment:
```suggestion
Note: The subscription queue can contain duplicate entries and is by design. This does not result in events to be published multiple times as publishers deduplicates at publishing time. Each time a publisher is restarted the subscription queue state will be rewritten and deduplicated.
```

Reply:
@mauroservienti it is incorrect, it doesn't dedupe during publishing, it dedupes during reading the subscriptions and when receiving a subscription request from a subscriber. Let me see if I can rewrite.